package.json

```diff
@@ -1,115 +1,117 @@
+	"main": "dist/index.js",
+	"scripts": {
+		"dev-ui": "NODE_ENV=development webpack-dev-server --config webpack.web.js --hot",
+		"build-ui": "NODE_ENV=development webpack --config webpack.web.js",
+		"dev-server": "nodemon ./build/test.js",
+		"watch-server": "webpack --config webpack.config.js --watch",
+		"build-server": "webpack --config webpack.config.js",
+		"watch-lib": "webpack --config webpack.lib.js --watch",
+		"build-lib": "webpack --config webpack.lib.js",
+		"watch-weblib": "webpack --config webpack.weblib.js --watch",
+		"build-weblib": "webpack --config webpack.weblib.js",
+		"watch-cli": "webpack --config webpack.bin.js --watch",
+		"build-cli": "webpack --config webpack.bin.js",
+		"dev": "NODE_ENV=development concurrently --kill-others-on-fail npm:watch-cli npm:watch-lib npm:watch-weblib npm:dev-ui",
+		"build": "NODE_ENV=production concurrently --kill-others-on-fail npm:build-cli npm:build-lib npm:build-weblib",
+		"start": "NODE_ENV=production npm run build && node ./build/server.js",
+		"lint": "eslint . --ext .ts,.tsx && prettier -c .",
+		"lint:fix": "eslint . --ext .ts,.tsx --fix && prettier -w . "
+	},
+	"author": "0xtsukino",
+	"license": "ISC",
+	"bin": {
+		"zkitter": "./bin/cli.js"
+	},
+	"types": "dist/src/index.d.ts",
+	"browser": "dist/browser.js",
+	"files": [
+		"dist/",
+		"readme.md"
+	],
+	"dependencies": {
+		"@nodeutils/defaults-deep": "^1.1.0",
+		"@waku/byte-utils": "^0.0.2",
+		"@waku/core": "^0.0.7",
+		"@waku/create": "^0.0.5",
+		"@waku/libp2p-utils": "^0.0.2",
+		"@waku/peer-exchange": "^0.0.1",
+		"@waku/proto": "^0.0.2",
+		"@zk-kit/identity": "^1.4.1",
+		"@zk-kit/incremental-merkle-tree": "^0.4.3",
+		"@zk-kit/protocols": "^1.11.1",
+		"abstract-level": "^1.0.3",
+		"async-mutex": "^0.3.1",
+		"bytewise": "^1.1.0",
+		"chalk": "^4.1.2",
+		"commander": "^10.0.0",
+		"elliptic": "^6.5.4",
+		"eventemitter2": "^6.4.9",
+		"isomorphic-fetch": "^3.0.0",
+		"js-waku": "^0.30.0",
+		"level": "^8.0.0",
+		"libp2p": "^0.41.0",
+		"module-alias": "^2.2.2",
+		"moment": "^2.29.4",
+		"nerd": "^0.0.0",
+		"promptly": "^3.2.0",
+		"readable-stream": "^4.3.0",
+		"readline": "^1.3.0",
+		"snarkjs": "^0.5.0",
+		"web3": "^1.3.5",
+		"web3-core": "^1.8.2",
+		"web3-eth-contract": "^1.8.2",
+		"webpack-dev-server": "^4.11.1",
+		"winston": "^3.3.3"
+	},
+	"devDependencies": {
+		"@istanbuljs/nyc-config-typescript": "^1.0.2",
+		"@types/bytewise": "^1.1.0",
+		"@types/elliptic": "^6.4.14",
+		"@types/node": "^14.14.37",
+		"@types/promptly": "^3.0.2",
+		"@types/sinon": "^10.0.11",
+		"@types/tape": "^4.13.2",
+		"@typescript-eslint/eslint-plugin": "^5.43.0",
+		"@typescript-eslint/parser": "^5.43.0",
+		"assert": "^2.0.0",
+		"browserify": "^17.0.0",
+		"browserify-zlib": "^0.2.0",
+		"concurrently": "^6.2.0",
+		"constants-browserify": "^1.0.0",
+		"copy-webpack-plugin": "^11.0.0",
+		"eslint": "^8.35.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-prettier": "^4.2.1",
```

Comment:
Related to previous [comment](https://github.com/zkitter/zkitter-js/pull/1/files#r1121978103).
I am not a fan of turning prettier rules into eslint rules, which is what `eslint-plugin-prettier` and `plugin:prettier/recommended` do. I prefer to just turn some linting rules do avoid clashing with eslint.
Which is what prettier recommends in their [docs](https://prettier.io/docs/en/integrating-with-linters.html#notes).

Reply:
Alright, sorry, wasn't aware of the configs repo!

I installed your plugins, but I can't get `eslint` to output something that makes sense. I get:
```
✖ 418 problems (284 errors, 134 warnings)
```

I'll dig into it a bit more and if I can't find why I'll push a wip commit so you can check it out.